docs/_cli_tutorials/hydration.md

```diff
@@ -30,15 +30,15 @@ The second reason is time. Your app's integration gets 30 seconds to run its API
 
 ## How Do I use Dehydration?
 
-Check out our example files app (https://github.com/zapier/zapier-platform-example-app-files) for an example of file dehydration in action with a working Zapier demo integration. You can even initialize a Zapier app based on that repo by entering ``zapier init . --template=files`` in Terminal.
+Check out our [example files app](https://github.com/zapier/zapier-platform-example-app-files) for an example of file dehydration in action with a working Zapier demo integration. You can even initialize a Zapier app based on that repo by entering ``zapier init . --template=files`` in Terminal.
 
 ***
 
 # Example Setup (optional)
 
 ## The Server
 
-We've got a very simple file server set up with Express at https://github.com/codebycaleb/express-files-example. The whole thing fits in one (hopefully) easy-to-read file, `index.js`. Let's take a quick look at what the API offers:
+We've got a very simple file server set up with Express at [https://github.com/codebycaleb/express-files-example](https://github.com/codebycaleb/express-files-example). The whole thing fits in one (hopefully) easy-to-read file, `index.js`. Let's take a quick look at what the API offers:
```

Comment:
Instead of using the URL as the link text, how about something like `in this GitHub repository` to ensure it's accessible?

Reply:
Tweaked text as per your suggestion! Thanks Jess